monkeytype/cli.py

```diff
@@ -274,7 +281,11 @@ def main(argv: List[str], stdout: IO, stderr: IO) -> int:
         return 1
 
     with args.config.cli_context(args.command):
-        handler(args, stdout, stderr)
+        try:
+            handler(args, stdout, stderr)
+        except HandlerError as err:
```

Comment:
This is the top level `main`, our final error boundary; is there any reason to be picky about exception types? Should this just be `except Exception as err` so we always return 1 on an exception and print it the same way?

Reply:
Generally in agreement with you. My thought was to treat `HandlerError` as an error type that signifies that something went wrong that doesn't represent a bug, whereas other exceptions would represent bugs in our code. I'd like to handle the latter case with a catch-all exception handler that writes the stacktrace to a file in `/tmp` so that users can include it in a bug report. That's obviously a larger change, so I'd prefer to make it in its own PR.